src/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp/Microsoft.DocAsCode.Metadata.ManagedReference.FSharp.fsproj

```diff
@@ -1,38 +1,32 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
+
   <Import Project="../Shared/base.props" />
   <PropertyGroup Condition=" '$(Configuration)' == 'Release' ">
     <DebugType>full</DebugType>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="AssemblyInfo.fs" />
-    <Compile Include="Fsc.fs" />
     <Compile Include="Log.fs" />
-    <Compile Include="FSharpCompilation.fs" />
     <Compile Include="FSharpProjectInfo.fs" />
+    <Compile Include="Fsc.fs" />
+    <Compile Include="FSharpCompilation.fs" />
     <Compile Include="FSharpProject.fs" />
   </ItemGroup>
 
+  <ItemGroup>
+    <PackageReference Include="Dotnet.ProjInfo" Version="0.9.0" />
+    <PackageReference Include="FSharp.Compiler.Service" Version="17.0.1" />
+    <PackageReference Include="FSharp.Compiler.Service.MSBuild.v12" Version="17.0.1" />
+  </ItemGroup>
+
   <ItemGroup>
     <ProjectReference Include="..\Microsoft.DocAsCode.Common\Microsoft.DocAsCode.Common.csproj" />
     <ProjectReference Include="..\Microsoft.DocAsCode.DataContracts.Common\Microsoft.DocAsCode.DataContracts.Common.csproj" />
     <ProjectReference Include="..\Microsoft.DocAsCode.DataContracts.ManagedReference\Microsoft.DocAsCode.DataContracts.ManagedReference.csproj" />
     <ProjectReference Include="..\Microsoft.DocAsCode.Metadata.ManagedReference.Common\Microsoft.DocAsCode.Metadata.ManagedReference.Common.csproj" />
-    <ProjectReference Include="..\Microsoft.DocAsCode.YamlSerialization\Microsoft.DocAsCode.YamlSerialization.csproj" />
     <ProjectReference Include="..\Microsoft.DocAsCode.Plugins\Microsoft.DocAsCode.Plugins.csproj" />
+    <ProjectReference Include="..\Microsoft.DocAsCode.YamlSerialization\Microsoft.DocAsCode.YamlSerialization.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
-    <PackageReference Include="Dotnet.ProjInfo" version="0.9.0" />
-    <PackageReference Include="FSharp.Compiler.Service" version="17.0.1" />
-    <PackageReference Include="FSharp.Compiler.Service.MSBuild.v12" version="17.0.1" />
-  </ItemGroup>
-
-  <ItemGroup Condition="'$(TargetFramework)' == 'net461' OR '$(TargetFramework)' == 'net46' OR '$(TargetFramework)' == 'net452'">
```

Comment:
It seems this section is the root cause, how about just removing this `<ItemGroup>` section?

Reply:
target framework section was removed, since we have them in the shared part.
And package reference section was moved in front of fsharp files section, maybe this is the reason...